rest-api-spec/src/main/resources/rest-api-spec/test/cat.shards/10_basic.yml

```diff
@@ -85,6 +85,7 @@
                     warmer.total_time                .+   \n
                     path.data                        .+   \n
                     path.state                       .+   \n
+                    docs.deleted                     .+   \n
```

Comment:
@rayshrey you may need to split this test case in two, since `docs.deleted` will be available only from 2.12.0